pkg/asset/agent/image/oc.go

```diff
@@ -323,13 +320,21 @@ func execute(executer executer.Executer, pullSecret, command string) (string, er
 	executeCommand := command[:] + " --registry-config=" + ps.Name()
 	args := strings.Split(executeCommand, " ")
 
-	stdout, stderr, exitCode := executer.Execute(args[0], args[1:]...)
+	var stdoutBytes, stderrBytes bytes.Buffer
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdout = &stdoutBytes
+	cmd.Stderr = &stderrBytes
 
-	if exitCode == 0 {
-		return strings.TrimSpace(stdout), nil
+	err = cmd.Run()
+	if err != nil {
```

Comment:
I think this should be:
`if err == nil {`

According to example https://pkg.go.dev/os/exec#Command

Reply:
With that change I was able to successfully install with `4.14.0-0.ci-2023-08-22-121303`

Reply:
Duplicate comments :-)